pydantic/mypy.py

```diff
@@ -760,15 +819,21 @@ def __init__(
         self.has_alias_generator = has_alias_generator
 
     def set_values_dict(self) -> dict[str, Any]:
```

Comment:
I think `get_values_dict` is a better name for this method.

What do you think?

Reply:
I'd rather agree. The current name looks like an accident to me.